models/credmark/protocols/credmark/vesting.py

```diff
@@ -138,11 +138,31 @@ def run(self, input: Account) -> AccountVestingInfo:
                     vesting_contract.functions.getClaimableAmount(input.address).call()
                 ))
             result.vesting_infos.append(vesting_info)
-            claims_all = [
-                dict(d['args']) for d in
-                vesting_contract.events.AllocationClaimed.createFilter(
-                    fromBlock=0, toBlock=self.context.block_number
-                ).get_all_entries()]
+            try:
+                allocation_claimed_events = vesting_contract.events.AllocationClaimed.createFilter(
```

Comment:
isn't this in a different pr?

Reply:
This was from another PR which shares some common files. I merged it here to avoid conflicts.